server/index.js

```diff
@@ -1,61 +1,69 @@
 const express = require('express');
+
 const app = express();
 const dataGenerator = require('./data-generator');
 const db = require('../database/index.js');
+const fs = require('fs');
+
+const generate = () => {
+  setInterval(() => {
+    const listings = dataGenerator.listingsGeneratorImport();
+    for (let i = 0; i < listings.length; i += 1) {
+      let content = listings[i];
```

Comment:
I *think* this can be a const because it is not being re-assigned in the loop iteration, which is the scope of this variable. But if the linter didn't complain, it's fine to leave as is.

Reply:
@christine-w fixed!